scripts/install/booklet.rdf

```diff
@@ -1,24 +1,47 @@
+        xml:base="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        >
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet">
+        <rdfs:label xml:lang="en-US"><![CDATA[Booklet]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Booklet]]></rdfs:comment>
+        <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#TAOObject"/>
+    </rdf:Description>
     <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#BookletFile">
-    <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
-    <rdfs:label xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:label>
-    <rdfs:comment xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:comment>
-    <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
-    <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#File"/>
-    <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#AsyncFile"/>
-    <tao:TAOGUIOrder><![CDATA[20]]></tao:TAOGUIOrder>
-  </rdf:Description>
-  
- </rdf:RDF>
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
+        <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#File"/>
+        <tao:TAOGUIOrder><![CDATA[20]]></tao:TAOGUIOrder>
+    </rdf:Description>
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Anonymous">
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Anonymous]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Indicate whether it should be anonymous]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
+        <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#RadioBox"/>
+        <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#Boolean"/>
+        <tao:TAOGUIOrder><![CDATA[30]]></tao:TAOGUIOrder>
+    </rdf:Description>
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOTest.rdf#QtiTestModel">
+        <rdf:type rdf:resource="http://www.tao.lu/Ontologies/TAOTest.rdf#TestModel"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[QTI 2.1 Test]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[]]></rdfs:comment>
+    </rdf:Description>
+
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOGroup.rdf#Booklets">
```

Comment:
We are inverting this property in other extensions. Please make the groups to belong to a booklet.